Userland/Libraries/LibWeb/CSS/CSSStyleDeclaration.h

```diff
@@ -26,9 +26,11 @@ class CSSStyleDeclaration : public Bindings::PlatformObject {
     virtual String item(size_t index) const = 0;
 
     virtual Optional<StyleProperty> property(PropertyID) const = 0;
+    virtual Optional<StyleProperty> custom_property(FlyString const& custom_property_name) const = 0;
 
-    virtual WebIDL::ExceptionOr<void> set_property(PropertyID, StringView css_text, StringView priority = ""sv) = 0;
-    virtual WebIDL::ExceptionOr<String> remove_property(PropertyID) = 0;
+    // property_name only needs to be passed for custom properties.
+    virtual WebIDL::ExceptionOr<void> set_property(PropertyID, StringView css_text, StringView property_name = ""sv, StringView priority = ""sv) = 0;
```

Comment:
A couple of points:

- Inserting `property_name` before `priority` feels like a footgun: Any existing calls to `set_property(PropertyID::Whatever, "something"sv, "important"sv)` will still compile, but will do something different than they did before. It's also odd to have the order different than the other `set_property()` method. Can we move it to the end?

- `Optional<StringView> property_name` seems clearer about it not always being provided. (Or even `Optional<FlyString const&>`, since that's how it's used.)

Reply:
I can go back to my previous implementation and move back property_name to the 2nd parameter and accept it as an Optional. Then pass an empty optional from wherever required.